organizations.json

```diff
@@ -1401,7 +1401,7 @@
         ]
     },
     {
-        "name": "Code for Raleigh",
+        "name": "Open Raleigh Brigade",
```

Comment:
Hey @jhibbets, can you confirm that the word "Brigade" is officially part of the name - and should display in all places next to "Open Raleigh"?

Reply:
Did any of the other web properties change as a result of the rename?